tests/test_to_dict.py

```diff
@@ -115,7 +114,7 @@ def test_to_dict(naip):
         "stac_version": pystac.get_stac_version(),
         "type": "Feature",
     }
-    assert result[0].to_dict() == expected
+    assert result[0].to_dict(transform_hrefs=False) == expected
```

Comment:
I found the network-touching tests by using `pytest --block-network`, and this one flagged up because of https://github.com/stac-utils/pystac/issues/960. This caused the slight change to the `expected` dict.